test/tests/components/Meter.js

```diff
@@ -0,0 +1,171 @@
+import expect from 'expect';
+import Meter, { Props } from '../../../src/meter';
+import { newComponent } from '../helpers';
+
+describe('Meter', function () {
+  describe('locals', () => {
+
+    const exampleProps = {
+      id: '12345',
+      className: 'fancy-class-name',
+      style: { margin: 10, position: 'relative' },
+      value: 25,
+      min: 0,
+      max: 50,
+      ranges: [
+        { startValue: 0, endValue: 50, fillingColor: 'green', labelColor: 'red' },
+        { startValue: 50, endValue: 80, fillingColor: 'yellow' },
+        { startValue: 80, endValue: 100 }
+      ],
+      baseLabelColor: '#000000',
+      baseFillingColor: '#ccc',
+      labelFormatter: () => 'test'
+    };
+
+    const componentMeter = new Meter({
+      ...exampleProps
+    });
+
+    it('should pass props', ()  => {
+      const {
+        id,
+        style,
+        value,
+        min,
+        max,
+        ranges,
+        baseLabelColor,
+        baseFillingColor
+      } = componentMeter.getLocals();
+      expect(id).toBe(exampleProps.id);
+      expect(style).toEqual(exampleProps.style);
+      expect(value).toBe(exampleProps.value);
+      expect(min).toBe(exampleProps.min);
+      expect(max).toBe(exampleProps.max);
+      expect(ranges).toEqual(exampleProps.ranges);
+      expect(baseLabelColor).toBe(baseLabelColor);
+      expect(baseFillingColor).toBe(baseFillingColor);
+    });
+    it('should compute className', ()  => {
+      const { className } = componentMeter.getLocals();
+      expect(className).toInclude('meter');
+      expect(className).toInclude('fancy-class-name')
+    });
+    it('should compute labelFormatter', ()  => {
```

Comment:
actually the props will test here is labelFormatter, formattedLabel is just the output